docs/userGuide/deployingTheSite.md

```diff
@@ -329,10 +329,185 @@ Here are the steps to set up Netlify:
 
 Now your site will be deployed on Netlify at the given address specified after deployment. It will be updated automatically when the default branch of your repo is updated.
 
-Additionally, **when contributors make a pull request to your GitHub repo, you can _preview_ the updated site** at the bottom of the pull request by clicking on `details` link in the PR:
+## Previewing Pull Requests for MarkBind sites
+
+<box type="info">
+
+This section is meant for users who are $$hosting their Markbind project on **Github**$$.
```

Comment:
let's keep this to 2nd person for consistency

Reply:
Okay, I have updated this to 2nd person